test/pageobjects/AddPublication.page.js

```diff
@@ -0,0 +1,44 @@
+const Page = require("./Page");
+
+class AddPublicationPage extends Page {
+
+    get heading() {
+        return $("//header//h6[text()='publications']");
```

Comment:
```suggestion
        return $("//h6");
```
There is only one tag h6 on the page